Application/Features/Commands/DeleteMovie/DeleteMovieCommandValidation.cs

```diff
@@ -0,0 +1,11 @@
+﻿using FluentValidation;
+
+namespace Application.Features.Commands.DeleteMovie;
+
+public class DeleteMovieCommandValidation : AbstractValidator<DeleteMovieCommand>
+{
+    public DeleteMovieCommandValidation()
+    {
+        RuleFor(x => x.Id).NotNull();
```

Comment:
What if I pass negative number, or zero?

Reply:
404 not found exception will be thrown